source/clear-linux/tutorials/nvidia.rst

```diff
@@ -139,11 +160,18 @@ Install the NVIDIA Linux Driver
       lsmod | grep ^nvidia
 
 
+#. Run a verification to restore system files that the NVIDIA installer likely
+   deleted.
+
+   .. code-block:: bash
+
+      sudo swupd verify --quick --fix --bundles=lib-opengl
+
 .. note::
 
-   The NVIDIA installer places files under the :file:`/usr` subdirectory which
-   are not managed by |CL| updates. The :command:`swupd verify --fix` command
-   should be avoided with the proprietary NVIDIA drivers in use.
+   The NVIDIA installer places some files under the :file:`/usr` subdirectory
+   which are not managed by |CL| updates. The :command:`swupd verify --fix`
+   command should be avoided with the proprietary NVIDIA drivers in use.
```

Comment:
Might want to explain more here why we're running verify with --bundles above.

Reply:
I think this line does that:
> Run a verification to restore system files that the NVIDIA installer likely
   deleted.

But I get your point - it's kind of conflicting that we say `swupd verify`, then you shouldn't `swupd verify`. I'll play with the wording a bit to see if it can be explained easier.

Reply:
I've pushed an update to the wording. Let me know what you think.